pkg/skyenv/values_darwin.go

```diff
@@ -6,13 +6,14 @@ package skyenv
 const (
 	//OS detection at runtime
 	OS = "mac"
-
 	// SkywirePath is the path to the installation folder.
 	SkywirePath = "/Library/Application Support/Skywire"
+	// Ptext is the description to use for the -p flag for config gen and for the visor
+	Ptext = "use mac installation path: " + SkywirePath
+	// Configjson is the config name generated by the script included with the installation on mac
+	Configjson = "skywire.json"
```

Comment:
What is the point of us introducing another name for the config file? We had `skywire-config.json` so far as default path.

Reply:
it was intentionally set like that to differentiate the config for the package from the default

Reply:
this was not introduced, it's been like this in the linux packages for over a year.

While it seems like a trivial thing to just use the default config name, it would require at this point another layer of config migration logic, and I have enough to do at this point for the packages and images to change that currently.